src/app/tab2/tab2.page.html

```diff
@@ -13,28 +11,22 @@
     </ion-toolbar>
   </ion-header>
 
-  <app-explore-container name="Tab 2 page"></app-explore-container>
-</ion-content>
-
-<ion-col size="6" *ngFor="let photo of photoService.photos; index as position">
-  <ion-img [src]="photo.webviewPath" (click)="showActionSheet(photo, position)"></ion-img>
-</ion-col>
-
-<ion-content>
-  <ion-grid>
-    <ion-row>
-      <ion-col size="6" *ngFor="let photo of photoService.photos; index as position">
-        <ion-img [src]="photo.webviewPath"></ion-img>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
-
   <ion-content>
+    <ion-grid>
+      <ion-row>
+        <ion-col
+          size="6"
+          *ngFor="let photo of photoService.photos; index as position"
+        >
+          <ion-img [src]="photo.webviewPath" ></ion-img>
+        </ion-col>
+      </ion-row>
+    </ion-grid>
+
     <ion-fab vertical="bottom" horizontal="center" slot="fixed">
       <ion-fab-button (click)="addPhotoToGallery()">
         <ion-icon name="camera"></ion-icon>
       </ion-fab-button>
     </ion-fab>
   </ion-content>
-  
```

Comment:
Too many spaces?

Reply:
Well usually in html there is no empty lines you can add them back if you feel it makes it easier to read

Reply:
Got ya! I'll remember that, readability is important, and I'll probably need to conform to a reading style that's best for everyone if I am to work on T3.